app/controllers/landing_page_controller.rb

```diff
@@ -0,0 +1,110 @@
+class LandingPageController < ActionController::Metal
+
+  class Denormalizer
+
+    def initialize(root: "composition", hooks: {})
+      @root = root
+      @hooks = hooks
```

Comment:
Yeah, that's the only thing they do at the moment. To be honest I didn't think too much about hooks, but just needed some way to add the proper path to the images.